reframe/core/buildsystems.py

```diff
@@ -351,8 +351,8 @@ class SingleSource(BuildSystem):
     #: automatically based on the extension of the source file.
     #: The automatically detected extensions are the following:
     #:
-    #:   - C: `.c`.
-    #:   - C++: `.cc`, `.cp`, `.cxx`, `.cpp`, `.CPP`, `.c++` and `.C`.
+    #:   - C: `.c`, `upc`.
+    #:   - C++: `.cc`, `.cp`, `.cxx`, `.cpp`, `.CPP`, `.c++`, and `.C`.
```

Comment:
I would skip the comma here for consistency with the rest.

Reply:
sounds good, done